### PR TITLE
[wip] jira helpers

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

**Comment by chrmod** [comment]
@joseph-cliqz I've posted your code here. please take a look if it is complete and working

File: vars/jira.groovy [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        changeString = " - No new changes"
+    }
+    return changeString
+}
+
+@NonCPS
+def getIssueList() {
+    def list = []
+    def changes = getChangeString()
+    def re = /AB-([0-9])*/
```
Comment (joseph-cliqz) [inline vars/jira.groovy]: The regular expression needs to be modified for each particular project.

File: vars/jira.groovy [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    }
+    if (!changeString) {
+        changeString = " - No new changes"
+    }
+    return changeString
+}
+
+@NonCPS
+def getIssueList() {
```
Comment (joseph-cliqz) [inline vars/jira.groovy]: def getIssueList(regularExpression) {

File: vars/jira.groovy (hunk 1 above)
Comment (joseph-cliqz) [inline vars/jira.groovy]: def re = regularExpression

**Pull request merged**